addon/mixins/route.js

```diff
@@ -1,6 +1,7 @@
 import Ember from 'ember';
 
 const { Mixin, RSVP, getOwner } = Ember;
+const PREFETCH_NOT_DEFINED = Object.freeze({});
```

Comment:
`Ember.__loader.require('ember-metal/empty_object').default` is too hacky IMO, so I just removed the `Object.freeze`.